content/references/protocol-reference/serialization.md

```diff
@@ -303,7 +303,10 @@ All such fields are serialized as the specific number of bits, with no length in
 ### Issue Fields
 [STIssue]: #issue-fields
 
-_(The "Issue" or "STIssue" type is part of multiple proposed extensions to the XRP Ledger protocol, including [XLS-30d: Automated Market Maker](https://github.com/XRPLF/XRPL-Standards/discussions/78) :not_enabled: and [Federated Sidechains](federated-sidechains.html) :not_enabled:)_ <!-- SPELLING_IGNORE: 30d -->
+<!--
+_(The "Issue" or "STIssue" type is part of multiple proposed extensions to the XRP Ledger protocol, including [XLS-30d: Automated Market Maker](https://github.com/XRPLF/XRPL-Standards/discussions/78) :not_enabled: and [Federated Sidechains](federated-sidechains.html) :not_enabled:)_ --> 
```

Comment:
Should the `Issue` section remain on xrpl-dev-portal? I don't think it's included in production (at least when I checked sField.cpp I didn't see it.)